src/snowflake/cli/api/console/console.py

```diff
@@ -78,11 +87,24 @@ def phase(self, enter_message: str, exit_message: Optional[str] = None):
             if exit_message:
                 self._print(self._format_message(exit_message, Output.PHASE))
 
+    @contextmanager
+    def indented(self) -> Iterator[Callable[[str], None]]:
+        """A context manager for indenting phase or step messages"""
```

Comment:
Docstring seems at odds with the exception raised in phase?

Reply:
Seems like `phase()` can't be used within `indented()`, but `indented()` can be used within `phase()`

Reply:
This was meant to mean something like "messages issues in steps and phases", but it reads horribly. Will rephrase.